### DISCUSSION TITLE
Group responsive class names?

### DISCUSSION THREAD

**Discussion by gustavojardim** [tailwindlabs/tailwindcss#13592] [Ideas]
Instead of writing: `md:w-full md:p-16 md:bg-indigo`
Do something like: `md:[w-full p-16 bg-indigo]`

**Comment by alex-krasikau** [comment]
Hey @gustavojardim!

This topic has been discussed multiple times, and it's unlikely that this feature will be added to Tailwind CSS. For more details, check out this [thread](https://twitter.com/adamwathan/status/1461519820411789314). The key point is that adding group variants would significantly increase the size of the CSS file.

**Comment by gustavojardim** [comment]
I think it could support both ways. It's not a big flaw (or it's not even one), but I think if that tailwind supported both, we would have better readability of classNames properties and it would be easier to maintain in certain scenarios. Anyway, I'm closing this topic.
